### PR TITLE
Added support any file beginning with psc*

### PR DESCRIPTION
**Pull request by doppioslash**
Used glob as you mentioned in #8.
Fix for #7

### CODE REVIEW

File: src/lib.js [hunk 1]
```diff
@@ -79,29 +80,15 @@ function downloadVersion(version, os) {
...
-            fs.chmodSync(path.join(destPath, 'psc-ide-server'), '0777');
-            fs.chmodSync(path.join(destPath, 'psc-publish'), '0777');
-            fs.chmodSync(path.join(destPath, 'psci'), '0777');
+function use (version) {
+  var srcPath = path.join(paths.PSVM_VERSIONS, version, 'purescript'),
+      destPath = path.join(paths.PSVM_CURRENT_BIN),
+      mg = new glob("**/psc*", {cwd: srcPath}, function (er, files) {
+        files = files.map(function(match) {
+            util.copy(path.join(srcPath, match), path.join(destPath, match));
+            fs.chmodSync(path.join(destPath, match), '0777');
```
Comment (ThomasCrevoisier) [inline src/lib.js:89]: Nice job ! 😃 
Just few comments : 
- `util.copy()` is async and return a promise, I would say that `fs.chmodSync()` should be done once the copy is made.
- By curiosity why are you creating an instance of `glob` ?

Comment (doppioslash) [inline src/lib.js:89]: Thanks :D
Last time I used Javascript extensively is in 2012.
I know the language, but a lot of stuff I've never used before (promises, etc).

I imagine I just copied that instance of `glob` out of some docs. If I remove that it gives me many fun errors.
I'm used to knowing my errors at compile time now, it's pretty cringeworthy to go back to Javascript's breaking at runtime and lack of info in the errors ^^;;
Feel free to fix this code as you want, I don't plan on learning node anyway, so this is as far as I can do without having to put in some learning time.

Comment (ThomasCrevoisier) [inline src/lib.js:89]: Yep no problem :)

I created a branch `7-support-psc-files`, could you change the target branch of you PR to this branch ? Then I'll be able to fix those little details ^^

Comment (doppioslash) [inline src/lib.js:89]: Cool, I created #21 with the right target branch :)

**Pull request closed without merging**